daemons/fenced/fenced_remote.c

```diff
@@ -2280,7 +2280,9 @@ fenced_process_fencing_reply(xmlNode *msg)
         } else {
             crm_notice("Action '%s' targeting %s using %s on behalf of %s@%s: "
                        "%s%s%s%s",
-                       op->action, op->target, device, op->client_name,
+                       op->action, op->target,
+                       ((device == NULL)? "any device" : device),
+                       op->client_name,
                        op->originator,
```

Comment:
might be more consistent to go with either "any device" or completely omit the "using" part in both places

Reply:
probably makes sense to omit it here